src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosJobFramework.scala

```diff
@@ -183,10 +187,26 @@ class MesosJobFramework @Inject()(
       )
       if (status == Protos.Status.DRIVER_RUNNING) {
         for (task <- tasks) {
-          runningTasks.put(task._2.name, new ChronosTask(task._3.getSlaveId.getValue))
-
-          //TODO(FL): Handle case if chronos can't launch the task.
-          log.info("Task '%s' launched, status: '%s'".format(task._1, status.toString))
+          //we create a TaskStatus object so that this task gets picked up in
```

Comment:
Block comment?

Also "any reconciliations occur" should be "any reconciliations occurring" or "any reconciliations that occur"